src/components/robot/pages/RobotDuplicatePage.tsx

```
@@ -0,0 +1,191 @@
+      saveButtonText={t("robot_duplication.buttons.duplicate")}
+      cancelButtonText={t("robot_duplication.buttons.cancel")}
+      isLoading={isLoading}
+    >
+      <>
+        <Box style={{ display: "flex", flexDirection: "column" }}>
+          {robot && (
+            <>
+              <span>{t("robot_duplication.descriptions.purpose")}</span>
+              <br />
+              <span
+                dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)